bigquery_etl/operational_monitoring/__init__.py

```diff
@@ -134,7 +136,7 @@ def _generate_sql(project, dataset):
             project, bucket, blob.name
         )
         normalized_slug = _bq_normalize_name(om_project["slug"])
-        render_kwargs.update({"branches": om_project["branches"]})
+        render_kwargs.update({"branches": om_project.get("branches", [])})
```

Comment:
Should we have some kind of assertion here that either `branches` or `pref` is defined, but (I assume) not both?

I think it's better for things to break in the case of unexpected input than to have unpredictable behaviour.

Reply:
good idea! I added an assertion. This is still pretty much in prototype phase where no one is actively using it but it will be handy in the future if people are trying to update the json files.